packages/admin-ui/src/components/Sidebar/components/SidebarDisclosure.tsx

```diff
@@ -50,7 +50,7 @@ export const SidebarDisclosure = forwardRef(function SidebarDisclosure(
         borderLeft: selectedFallback ? '$mainSelected' : 'solid transparent',
         borderRight: 'solid transparent 3px',
         borderLeftWidth: '3px',
-        zIndex: 9999,
+        zIndex: 99,
```

Comment:
Do we have any zIndex token to be used here?

Reply:
any of the zIndex are defined yet, that's why I preferred to go with a hardcode one